src/Symfony/Component/Serializer/Exception/AggregatedException.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Exception;
+
+final class AggregatedException extends RuntimeException implements AggregableExceptionInterface
+{
+    /**
+     * @var bool
+     */
+    public $has = false;
+
+    /**
+     * @var array
+     */
+    private $collection;
+
+    public function __construct(\Throwable $previous = null)
+    {
+        parent::__construct('Errors occurred during the denormalization process', 0, $previous);
+    }
+
+    public function add(string $param, AggregableExceptionInterface $exception): self
+    {
+        $this->has = true;
+        $this->collection[] = [$param, $exception];
+
+        return $this;
+    }
+
+    /**
+     * @return \Generator<string, AggregableExceptionInterface>
+     */
+    public function getExceptions(): \Generator
```

Comment:
According to conventions and to go along with the `add()` or `set()` it should be `all()`:
```suggestion
    public function all(): \Generator
```

Reply:
fixed